src/network/networkd-address.c

```diff
@@ -658,9 +664,100 @@ int address_configure(
                 return log_link_error_errno(link, r, "Could not add address: %m");
         }
 
+        if (address->ipv4_duplicate_address_detection) {
+                if (DEBUG_LOGGING) {
+                        _cleanup_free_ char *pretty = NULL;
+
+                        (void) in_addr_to_string(AF_INET, &address->in_addr, &pretty);
+                        log_debug("Starting IPv4ACD client. Probing address '%s'", strna(pretty));
+                }
+
+                r = sd_ipv4acd_start(address->acd);
+                if (r < 0)
+                        log_link_warning_errno(link, r, "Failed to start IPv4ACD client, ignoring: %m");
+        }
+
         return 1;
 }
 
+static void static_address_on_acd(sd_ipv4acd *acd, int event, void *userdata) {
+        _cleanup_free_ char *pretty = NULL;
+        Address *address;
+        Link *link;
+        int r;
+
+        assert(acd);
+        assert(userdata);
+
+        address = (Address *) userdata;
+        link = address->link;
+
+        (void) in_addr_to_string(AF_INET, &address->in_addr, &pretty);
+        switch (event) {
+
+        case SD_IPV4ACD_EVENT_STOP:
+                log_link_debug(link, "Stopping ACD client ...");
```

Comment:
No space before the ellipsis. (Right now it looks like the ellipsis is a placeholder for a client name.)